src/app/utils/random.ts

```diff
@@ -0,0 +1,3 @@
+export function randomBetween(min = 0, max = 10) {
```

Comment:
Add what type it returns.

Also you can add type to min and max. I'm not sure we use this function it will force us to set numbers when we don't force it by adding types